src/objects/tests/commands/test_setup_configuration.py

```diff
@@ -1,123 +0,0 @@
-from io import StringIO
-
-from django.contrib.sites.models import Site
-from django.core.management import CommandError, call_command
-from django.test import TestCase, override_settings
-from django.urls import reverse
-
-import requests_mock
-from rest_framework import status
-from zgw_consumers.client import build_client
-from zgw_consumers.models import Service
-
-from objects.config.demo import DemoUserStep
-from objects.config.objecttypes import ObjecttypesStep
-from objects.config.site import SiteConfigurationStep
-
-from ..utils import mock_service_oas_get
-
-
-@override_settings(
-    SITES_CONFIG_ENABLE=True,
-    OBJECTS_DOMAIN="objects.example.com",
-    OBJECTS_ORGANIZATION="ACME",
-    OBJECTS_OBJECTTYPES_CONFIG_ENABLE=True,
-    OBJECTTYPES_API_ROOT="https://objecttypes.example.com/api/v2/",
-    OBJECTTYPES_API_OAS="https://objecttypes.example.com/api/v2/schema/openapi.yaml",
-    OBJECTS_OBJECTTYPES_TOKEN="some-random-string",
-    DEMO_CONFIG_ENABLE=True,
-    DEMO_TOKEN="demo-random-string",
-    DEMO_PERSON="Demo",
-    DEMO_EMAIL="demo@demo.local",
-)
-class SetupConfigurationTests(TestCase):
```

Comment:
Not sure if we want a similar test for the new setup. It might feel a bit excessive but I guess it does verify that all expected steps are executed and work together when used with an example.yaml file

@swrichards what do you think about this?